cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -134,8 +134,6 @@ if (EMSCRIPTEN_FORCE_COMPILERS)
 	set(CMAKE_CXX_COMPILER_ID Clang)
 	set(CMAKE_CXX_STANDARD_COMPUTED_DEFAULT 98)
 
-	set(CMAKE_C_COMPILER_TARGET "asmjs-unknown-emscripten")
```

Comment:
If the only difference is the invocation of clang, then `--triple` is unnecessary even in the asmjs case. But I'm not sure if `CMAKE_C_COMPILER_TARGET` also has some other effect on CMake that I don't know about.